ab_tool/tests/test_experiment_pages.py

```diff
@@ -95,10 +95,10 @@ def test_submit_create_experiment(self):
         experiment = {
                 "name": "experiment", "notes": "hi", "uniformRandom": True,
                 "csvUpload": False,
-                "tracks": [{"id": None, "weighting": None, "name": "A"}]
+                "tracks": [{"id": None, "weighting": None, "name": "A", "resource_link_id": self.resource_link_id}]
```

Comment:
Should this match the `experiment` declaration on line 111, below? I'm not sure an element of `tracks` needs to have this key.